classes/submission/Submission.inc.php

```diff
@@ -1157,6 +1157,12 @@ function isCCLicense() {
 		return preg_match('/creativecommons\.org/i', $this->getLicenseURL());
 	}
 
+	/**
+	 * @copydoc DataObject::getDAO()
+	 */
+	function getDAO() {
+		return Application::getSubmissionDAO();
```

Comment:
At some point it would be nice to find a way of removing these functions from `Application` -- but that's a subject for another time.